### PR TITLE
Enable Brand colours to be updated

### PR DESCRIPTION
**Pull request by sareh**
No ticket

**Overall change:** Brand colours are now being passed from the Service config to `psammead-brand`, allowing for services other than News/WorldService.  See related PRs: https://github.com/bbc/psammead/pull/2127 https://github.com/bbc/psammead/pull/2120

**Code changes:**

-  Add brandBackgroundColour & brandLogoColour values to services config, under the new `theming` key
- Use brandBackgroundColour & brandLogoColour values from Service Context to map to `backgroundColour` & `logoColour` props to `psammead-brand`.
- Since using the latest `psammead-brand@5`, am also pulling in all the other updated Psammead package dependencies.

**There should be no visual regression - this PR is to pass colours as props to the brand component**

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by RayNjeri — COMMENTED** [review]
LGTM but am guessing this needs to be merged first https://github.com/bbc/psammead/pull/2127/ and also a package bump is required for the changes to be effected.

**Review by RayNjeri — APPROVED** [review]

**Review by HarveyPeachey — APPROVED** [review]
LGTM

File: src/app/lib/config/services/amharic.js [hunk 1]
```diff
@@ -38,6 +38,10 @@ const service = {
     manifestPath: '/manifest.json',
     swPath: '/sw.js',
     frontPageTitle: 'ዜና',
+    theming: {
+      brandBackgroundColour: `${C_POSTBOX}`,
```
Comment (DenisHdz) [inline src/app/lib/config/services/amharic.js:42]: Are we going to apply different styles for other components in the future? If so, should we extend the `theming` and add another key with the component name like:
```
theming: {
      brand: {
            backgroundColour: `${C_POSTBOX}`,
            logoColour: `${C_WHITE}`, 
      },
      ...
}

```

Comment (sareh) [reply src/app/lib/config/services/amharic.js:42]: I think there'll only be a couple of things - brand & navigation, so it shouldn't need extra nesting.

**Review by DenisHdz — APPROVED** [review]

**Review by chris-hinds — APPROVED** [review]

**Pull request merged**